src/components/tables/TableActions.ts

```diff
@@ -49,3 +52,12 @@ export const modifyState = (
     shouldResetPage,
   },
 });
+
+export const updateSelectedRows = (id: string, selectedIds: string[], hasMultipleSelectedRow: boolean): IReduxAction<ITableActionPayload> => ({
```

Comment:
i would suggest to write `hasMultipleSelectedRow: boolean = false` since we won't always need to this feature on the table